hypervisor/arch/x86/page.c

```diff
@@ -65,3 +65,132 @@ const struct memory_ops ppt_mem_ops = {
 	.get_pdpt_page = ppt_get_pdpt_page,
 	.get_pd_page = ppt_get_pd_page,
 };
+
+#define EPT_ADDRESS_SPACE(size)	((size != 0UL) ? (size + CONFIG_PLATFORM_LO_MMIO_SIZE) : 0UL)
```

Comment:
Add a comment for this MACRO:  "The size of the guest physical address space, covered by the EPT page table of a VM"